gammapy/detect/test_statistics.py

```diff
@@ -196,7 +319,7 @@ def _ts_value(position, counts, exposure, background, kernel, flux,
         Background map.
     exposure : array
         Exposure map.
-    kernel : `astropy.convolution.Kernel2D`
+    kernel : astropy.convolution.core.Kernel2D
```

Comment:
Why did you change this?
I think the old version is the correct one that creates a HTML link and the new one doesn't?
(see https://gammapy.readthedocs.org/en/latest/development/index.html#where-should-i-import-from)